libs/api/shared/src/index.ts

```diff
@@ -1 +1,2 @@
 export { default as KordisRequest } from './lib/models/request.model';
+export { KordisGqlContext } from './lib/models/request.model';
```

Comment:
How about removing the default export and simply exporting `*` from `./lib/models/request.model`?